docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst

```diff
@@ -105,9 +105,14 @@ Findable
 		
 	- Authorship, contact information, and description:
 	
-		- You can add author and description metadata to your descriptor file. Adding an author will make it selectable on the Author facet in Dockstore’s search and a description helps because the text search uses it as one of the fields to sift through. 
+		- You can add author and description metadata to your descriptor file. Adding an author will make it selectable on the Author facet in Dockstore’s search and a description helps because the text search uses it as one of the fields to sift through.
+		
+	- Link GitHub repository:
+	
+		- Additionally, you can include a link to your original GitHub repo in order to connect the Dockstore entry with the source code. If your workflow is moved out of Dockstore, this can help to keep it connected with the original source code. You may consider doing so especially when you have additional comprehensive README files available on GitHub.
```

Comment:
"Link github repo" is a bit confusing here. I think we are suggested they paste their GitHub repo readme URL in the description metadata section of the WDL, correct? I may clarify the language for that.